tutorials/math/random_number_generation.rst

```diff
@@ -226,6 +394,38 @@ floating-point number between 0.0 and 1.0. We can use this to create a
             # 5% chance of being returned.
             return "Rare"
 
+ .. code-tab:: csharp
+
+    public override void _Ready()
+    {
+        GD.Randomize();
+
+        for (int i = 0; i < 100; i++)
+        {
+            GD.Print(get_item_rarity());
+        }
+    }
+
+    public string get_item_rarity()
```

Comment:
Similarly to `get_fruit`, this method isn't PascalCase and also could be renamed. GenerateItemRarity? CalculateItemRarity?